src/commands/lambda/flare.ts

```diff
@@ -379,14 +456,28 @@ export const getLogStreamNames = async (cwlClient: CloudWatchLogsClient, logGrou
  * @param cwlClient
  * @param logGroupName
  * @param logStreamName
+ * @param startMillis
+ * @param endMillis
  * @returns the log events or an empty array if no log events are found
  * @throws Error if the log events cannot be retrieved
  */
-export const getLogEvents = async (cwlClient: CloudWatchLogsClient, logGroupName: string, logStreamName: string) => {
-  const command = new GetLogEventsCommand({
+export const getLogEvents = async (
+  cwlClient: CloudWatchLogsClient,
+  logGroupName: string,
+  logStreamName: string,
+  startMillis: number,
+  endMillis: number
+) => {
+  const config: any = {
     logGroupName,
     logStreamName,
-  })
+    limit: MAX_LOG_EVENTS_PER_STREAM,
+  }
+  if (startMillis !== 0 || endMillis !== 0) {
+    config.startTime = startMillis
+    config.endTime = endMillis
+  }
```

Comment:
Unlike `DescribeLogStreamsCommand`, `GetLogEventsCommand` does accept optional `startTime` and `endTime` config values